bin/common.sh

```diff
@@ -65,8 +65,9 @@ function get_k8s_version() {
 }
 
 function otomi_image_tag() {
-  local otomiVersion=$([ -n "${CLOUD+x}${CLUSTER+x}" ] && yq r $clusters_file "clouds.$CLOUD.clusters.$CLUSTER.otomiVersion")
-  [ -n "$otomiVersion" ] && echo $otomiVersion || echo 'latest'
+  local otomi_version=$([ -n "${CLOUD+x}${CLUSTER+x}" ] && yq r $clusters_file "clouds.$CLOUD.clusters.$CLUSTER.otomiVersion")
+  [ -n "$otomi_version" ] && otomi_version='latest'
```

Comment:
This means: if `$otomi_version` is non-zero length `otomi_version='latest'`. That is not correct...

Reply:
is there no test for that? can you add that?

Reply:
#392